src/core/public/core_system.ts

```diff
@@ -146,6 +146,7 @@ export class CoreSystem {
   private reportKibanaLoadedEvent(analytics: AnalyticsServiceStart) {
     analytics.reportEvent('Loaded Kibana', {
       kibana_version: this.coreContext.env.packageInfo.version,
+      protocol: window.location.protocol,
```

Comment:
Nope, I don't think so.
The protocol does not change, so having this piece of data once is enough.

Reply:
sure! I meant if it's relevant for other events like `dashboard-loaded`. If we set it as a context, it'll enrich all the events instead of having to correlate the `dashboard-loaded` event with the `Loaded Kibana` one to figure out the protocol.